packages/spec/package.json

```diff
@@ -23,15 +23,17 @@
   },
   "scripts": {
     "prebuild": "rimraf dist && mkdir dist",
-    "build": "tsc && node ../../esbuild.js mosaic-spec",
+    "build": "tsc && node ../../esbuild.js mosaic-spec && npm run schema",
     "lint": "eslint src test",
+    "schema": "ts-json-schema-generator -f tsconfig.json -p src/spec/Spec.ts -t Spec --no-type-check --no-ref-encode --functions hide > dist/mosaic-schema.json",
```

Comment:
FWIW this fails on a fresh clone of the repo bc the `dist/` directory doesn't exist

Reply:
Thanks! Yeah, this is still WIP, but also `schema` is not meant to typically be run in isolation; it's now tied in to both the prebuild and pretest hooks.